CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.10.3
+
+### New features
+
+* Add support for Kafka message headers, available through the `$kafka_headers` metadata variable.
```

Comment:
Can we use `$kafka.headers` (a nested object) for the headers in the metadata? The reason is that our goal is to properly namespace all onramp/offramp specific metadata with the onramp/offramp id, so we dont pollute the keyspace here and we are free to add additional metadata values in a clean way into the `$kafka` object next to the `headers`.

Reply:
I know this introduces an inconsistency between `$kafka_key` and `$kafka_headers`. Feel free to also put the `key` into the `$kafka` object in this PR, otherwise we tackle this with a follow up PR.

Reply:
So I was thinking of having the variable as `$kafka_headers` in this PR, and then in a follow-up PR, we could have both `$kafka_key` and `$kafka_headers` inside the `$kafka` variable, which can then be used through `$kafka.headers` and `$kafka.key`, since the issue that PR is tackling is mainly related to message headers.
If it's better to have this change in this PR itself, I'll update it accordingly, no issues! :)

Reply:
That is also fine, keeping the PRs small.